src/main/scala/io/iohk/ethereum/crypto/package.scala

```diff
@@ -45,6 +46,24 @@ package object crypto {
     generator.generateKeyPair()
   }
 
+  /** @return (privateKey, publicKey) pair */
+  def keyPairToByteArrays(keyPair: AsymmetricCipherKeyPair): (Array[Byte], Array[Byte]) = {
```

Comment:
Minor detail: We currently have a very similar method `AsymmetricCipherKeyPairSerializable.toHexStrings`, should we keep only one of them?